src/ui/components/AddressBadge/AddressBadge.tsx

```diff
@@ -27,7 +27,10 @@ export function AddressBadge({
       <WalletAvatar address={wallet.address} size={14} borderRadius={4} />
       <UIText
         kind="caption/regular"
-        style={{ overflow: 'hidden', textOverflow: 'ellipsis' }}
+        style={{
+          overflow: 'hidden',
+          textOverflow: 'ellipsis',
+        }}
```

Comment:
unnecessary diff

Reply:
reverted, yeah, just removing the line didn't work :)